bleak/backends/corebluetooth/PeripheralDelegate.py

```diff
@@ -87,14 +87,21 @@ def futures(self) -> Iterable[asyncio.Future]:
     @objc.python_method
     async def discover_services(self, use_cached: bool = True) -> NSArray:
         if self._services_discovered_future.done() and use_cached:
```

Comment:
I think we should be able to get rid of the future done check here.


```suggestion
        if use_cached:
```